README.md

```diff
@@ -27,10 +27,10 @@ Settings view (<kbd>cmd-,</kbd>).
 
 You can customize matching pairs in Bracket Matcher at any time. You can do so either Globally via the Settings view (<kbd>cmd-,</kbd>) or at the Scope level via `config.cson` (<kbd>cmd-shift-p</kbd> + "config"). Changes take effect immediately.
```

Comment:
Globally and Scope shouldn't be capitalized.  Settings View should be though.

Also,
> ...or at the scope level via your `config.cson`.

You don't need to include the shortcuts.  I think it's safe to assume that if they're reading this they should know how to get to the Settings View and config.cson.